pkg/kn/commands/service/describe_test.go

```diff
@@ -546,6 +547,20 @@ func createTestService(name string, revisionNames []string, conditions duckv1.Co
 		}
 		service.Status.Traffic = trafficTargets
 	}
+
+	if len(serviceaccount) > 0 {
```

Comment:
This check reads odd... if `serviceaccount` is suppose to be the `serviceAccountName` as it's set in the `PodSpec` can we call it that and also check for `!= ""` instead?

Reply:
Fixed. Thanks.